ush/parsing_namelists_FV3.sh

```diff
@@ -443,25 +443,21 @@ local SKEB_LSCALE=${SKEB_LSCALE:-"-999."}
 local SKEBNORM=${SKEBNORM:-"1"}
 local SKEB_NPASS=${SKEB_NPASS:-"30"}
 local SKEB_VDOF=${SKEB_VDOF:-"5"}
-local SKEBINT=${SKEBINT:-"1800"}
+local SKEBINT=${SKEBINT:-"0"}
 local SHUM=${SHUM:-"-999."}
 local ISEED_SHUM=${ISEED_SHUM:-${ISEED}}
 local SHUM_TAU=${SHUM_TAU:-"-999."}
 local SHUM_LSCALE=${SHUM_LSCALE:-"-999."}
-local SHUMINT=${SHUMINT:-"3600"}
+local SHUMINT=${SHUMINT:-"0"}
 local SPPT=${SPPT:-"-999."}
 local ISEED_SPPT=${ISEED_SPPT:-${ISEED}}
 local SPPT_TAU=${SPPT_TAU:-"-999."}
 local SPPT_LSCALE=${SPPT_LSCALE:-"-999."}
 local SPPT_LOGIT=${SPPT_LOGIT:-".true."}
 local SPPT_SFCLIMIT=${SPPT_SFCLIMIT:-".true."}
 local USE_ZMTNBLCK=${use_zmtnblck:-".true."}
-local SPPTINT=${SPPTINT:-"1800"}
+local SPPTINT=${SPPTINT:-"0"}
 local PBL_TAPER=${pbl_taper:-"0,0,0,0.125,0.25,0.5,0.75"}
-local OCNSPPT=${OCNSPPT:-"0.8,0.4,0.2,0.08,0.04"}
```

Comment:
https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/global_control.nml.IN#L432-L43 

While I haven't tested - I'm pretty sure removing these will make things crash: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/global_control.nml.IN#L432-L43